src/screens/Requests/Requests.tsx

```diff
@@ -143,7 +141,7 @@ const Requests = (): JSX.Element => {
   };
   /* istanbul ignore next */
```

Comment:
remove this comment and add test for this code too.

Reply:
@pranshugupta54

Reply:
Ohk

Reply:
@aashimawadhwa, I've removed the ignore part for 3 codes. But I'm not able to write tests for `loadMoreRequests` function as it's a major bug in InfiniteScroll package which gives issue for loading new data on scroll on different sizes. (It's not being maintained for 3 years+)